exercises/functions/functions1.rs

```diff
@@ -1,8 +1,7 @@
 // functions1.rs
 // Execute `rustlings hint functions1` or use the `hint` watch subcommand for a hint.
 
-// I AM NOT DONE
-
 fn main() {
+    fn call_me() {};
```

Comment:
See I did the same thing but I suspect they wanted me to define the fn outside of main.

Whats super nice though is that its just a jmp scope so who cares :)

Reply:
Right? I'm loving how easy it is to scope functions like that.